doc/nrf/libraries/fem_al/index.rst

```diff
@@ -0,0 +1,48 @@
+.. _fem_al_lib:
+
+FEM Abstraction Layer
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The library is a wrapper for the MPSL FEM.
+
+Overview
+********
+
+The library is designed to simplify low level usage of a FEM hardware.
+
+Requirements
+************
+
+The library requires the usage of a FEM and MPSL functionality.
+
+Configuration
+*************
+
+The library is enabled by the :kconfig:option:`CONFIG_FEM_AL_LIB`.
```

Comment:
```suggestion
To enable this library, use the Kconfig option ``CONFIG_FEM_AL_LIB``.
```
As this option is not available in the Kconfig reference, you should not use the :kconfig:option: tag.